tests/baselines/reference/propertyParameterWithQuestionMark.js

```diff
@@ -3,11 +3,11 @@ class C {
     constructor(public x?) { }
 }
 
-// x should not be an optional property
-var v: C = {}; // Should fail
+// x should be an optional property
+var v: C = {}; // Should succeed
 var v2: { x? }
-v = v2; // Should fail
-var v3: { x } = new C; // Should succeed
+v = v2; // Should succeed
+var v3: { x } = new C; // Should fail

```

Comment:
Every time these comments need to be updated, I wonder if they ever should have been there to begin with. You have already have the error messages.